### DISCUSSION TITLE
External REST APIs

### DISCUSSION THREAD

**Discussion by kumarkeviv** [Budibase/budibase#4883] [Help] [answered]
Can I call an external REST API running on a different server from Budibase?

My usecase is, whenever a row is updated with a particular value in a column for example the status, a REST API should be invoked which reads the value of the row and calls the API with that body.

The REST API has already business logic and integeration, so this would be very helpful.

Can this be done already?

**Comment by mjashanks** [comment, marked as answer]
Yes - this is possible. You can add a new data source from the `Data` section: https://docs.budibase.com/docs/rest
